packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -923,4 +881,70 @@ export default class Editor {
     }
 
     //#endregion
+
+    //#region Dark mode APIs
+
+        /**
+     * Set the dark mode state and converts the content.
+     * @param desiredDarkMode The desired status of dark mode. True if the editor should be in dark mode, false if not.
+     */
+    public setDarkModeState(desiredDarkMode?: boolean) {
+        if (this.isDarkMode() != desiredDarkMode) {

```

Comment:
Personally I prefer early exits for methods that are 100% inside an if statement but up2u

Reply:
Yeah, it's a style thing. Save an indent, save a life.